app/helpers/eitc_zendesk_instance.rb

```diff
@@ -87,7 +87,7 @@ class EitcZendeskInstance
     EIP_STATUS_ID_UPLOAD => "Reached ID upload page",
     EIP_STATUS_SUBMITTED => "Submitted EIP only form",
     EIP_STATUS_READY_FOR_PHONE_CALL => "Ready for phone call",
-    EIP_STATUS_PREP_IN_PROGRESSS => "EIP prep in progress",
+    EIP_STATUS_PREP_IN_PROGRESS => "EIP prep in progress",
```

Comment:
Thanks for catching this! Do we use that misspelled constant anywhere?

Reply:
No, these were the only direct usages in the application.